Automattic/themes
# Blockbase: Clear cache when changing global styles from the Customizer

#### Changes proposed in this Pull Request:
Global Styles are cached in a transient, here:
https://github.com/WordPress/gutenberg/blob/f9eb4065cbbebef444ce5f53e7ad708146a2501e/lib/global-styles.php#L40

This deletes the transient when changes are made to Global Styles so that the newest version is loaded when you view the frontend.

To test:
- Go to the customizer and change fonts/colors
- Load the frontend
- Confirm you see your changes immediately

#### Related issue(s):
#4173
